app/controllers/hint_controller.rb

```diff
@@ -2,6 +2,10 @@ class HintController < ApplicationController
   def hint
     return if params[:q].blank?
     @hint = Hint.match(params[:q])
+    if (@hint && ENV['GOOGLE_ANALYTICS'])
```

Comment:
I'll get back to you shortly on what we might care about in terms of testing for this block.